src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -1,16 +1,15 @@
-using Dynamo.Utilities;
-using Dynamo.Logging;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Diagnostics;
-using Dynamo.Configuration;
-using DynamoUtilities;
+using Dynamo.Logging;
+using Dynamo.Utilities;
 
 namespace Dynamo.Search
 {
+    /// The search API in this class has been replaced with the Lucene search.
+
     /// <summary>
-    ///     A dictionary of objects for search
+    ///     A dictionary of objects.
```

Comment:
Keeing this class because we are still using the SearchEntries dictionary and we don't have a dictionary object in Lucene. All the search related methods have been removed.

Reply:
Thanks for the clarification, can we rename this class to not include search in the class name?

Reply:
yes done.